src/wrapping.rs

```diff
@@ -63,6 +70,68 @@ impl<T: ConstantTimeEq> ConstantTimeEq for Wrapping<T> {
     }
 }
 
+#[cfg(feature = "num-traits")]
+impl<T: Zero> num_traits::Zero for Wrapping<T>
+    where Wrapping<T>:
+    Add<Wrapping<T>, Output = Wrapping<T>>
+    + PartialEq<Wrapping<T>>
+{
+    fn zero() -> Self {
+        Self::ZERO
+    }
+
+    fn is_zero(&self) -> bool { self == &Self::ZERO }
+}
+
+#[cfg(feature = "num-traits")]
+impl<T: Integer> num_traits::One for Wrapping<T>
+    where Wrapping<T>:
+    Add<Wrapping<T>, Output = Wrapping<T>>
+    + Mul<Wrapping<T>, Output = Wrapping<T>>
+{
+    fn one() -> Self {
+        Wrapping(T::ONE)
+    }
+}
+
+#[cfg(feature = "num-traits")]
+impl<T: Zero> Div<Wrapping<T>> for Wrapping<T>
+    where Wrapping<T>: Div<NonZero<T>, Output = Wrapping<T>>
+{
+    type Output = Wrapping<T>;
+
+    fn div(self, rhs: Wrapping<T>) -> Self::Output {
```

Comment:
This and `rem` panics in case of division by zero.
This is the intended behavior of this crate, but I'm not sure that "this is the right panic" - i.e. now we'll panic on `unwrap()`ping of `NonZero` on zero, and the panic message might not be ideal; should I instead `assert_ne(rhs, T::Zero)` instead?